app/services/jokes.js

```diff
@@ -0,0 +1,12 @@
+const axios = require('axios');
+
+const apiURL = 'https://geek-jokes.sameerkumar.website/api?format=json';
```

Comment:
This is a variable constant, so when we have a variable constant the best practice is to create it into a constants path and constants file that it could be located into de app directory. At the same level for example of helpers, interactor, models, etc. 

If you don't understand me, tell me and we will see.